### ISSUE TITLE
[Serious] Page is missing skip link, headings or landmarks

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#648]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87517 |
| **Checkpoint** | 2.4 Navigable › Skip Navigation, Headings or Landmarks (2.4.1.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 06 Dashboard |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/04/26 at 8:40 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The page does not have at least one of: a skip link, proper heading structure, HTML5 sectioning elements, or ARIA landmarks.
Global: there isn't a link at the top of the app to allow users to skip the menu and jump to the main content area.  See Settings User Experience video at 1:00.

## Recommendation

Rule : 
A method MUST be provided to skip navigation and other page elements that are repeated across web pages.

How To Fix: 
Fix this issue by using ONE or MORE of the following techniques:
1. Provide a "skip to main content" link at the top of each page that goes directly to the main content area. This technique is preferred because it helps both screen reader users and keyboard users.
2. Use good semantic heading structure, preferably using an <h1> element to start the main content area.
3. Use HTML5 sectioning elements to mark sections of the layout. Place the main content in an HTML5 <main> section.
4. Use ARIA landmark attributes to mark sections of the layout. Place the main content a container marked with role="main".

Reference: 
Creating a Skip to Content link: https://www.makethingsaccessible.com/guides/creating-a-skip-to-content-link/
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/page-structure/

Background: 
In contrast to a sighted person's ability to visually skip past repeated content at the top of a page - such as heading graphics and navigation links - people who are blind must read content sequentially with a screen reader, starting at the top of the page. Similarly, mouse users can click directly on an element in the middle of a page, while keyboard users must tab past all links, buttons, form fields, etc. to get to an element further into a page. To help screen reader and keyboard users bypass content repeated across multiple pages, a mechanism is required - such as a skip link, good heading structure, HTML5 sectioning elements, or ARIA landmarks. This allows screen reader and keyboard users to more efficiently get to the main content of a page.
